test/integration/faraday_middleware_test.rb

```diff
@@ -22,6 +22,30 @@ def teardown
       Circuitbox.reset
     end
 
+    def open_circuit
```

Comment:
Yes you are right, we should move that to the `integration_helper.rb` I think as it is very useful, and should be part of the helper suite. @chiefcll do you fancy moving it? Otherwise I'm happy to do it after merge as well.